### DISCUSSION TITLE
PHPMailer version

### DISCUSSION THREAD

**Discussion by Aleph-Design** [PHPMailer/PHPMailer#2576] [General]
in the file: 'PHPMailer.php' I see:
"
/**
 * PHPMailer - PHP email creation and transport class.
 * PHP Version 5.5.
"
Is that still true?

**Comment by Synchro** [comment]
Yes: the minimum version of PHP supported by the current version of PHPMailer is PHP 5.5. It will also work fine with all other versions up to (and including) PHP 8.1. That particular comment is from an old convention used by PEAR to designate supported PHP versions which has largely been superseded by composer, but it's still used by some legacy systems.
